app/src/main/java/io/homeassistant/companion/android/sensors/NotificationSensorManager.kt

```diff
@@ -70,7 +71,11 @@ class NotificationSensorManager : NotificationListenerService(), SensorManager {
     }
     override fun hasSensor(context: Context): Boolean {
         val uiManager = context.getSystemService<UiModeManager>()
```

Comment:
You can move this down into the `if`, we don't need the UI manager if this is Automotive

Reply:
thanks :)